scripts/mkcloud

```diff
@@ -151,6 +151,7 @@ function show_environment()
     echo "-------------------------------"
     echo " cloudsource: $cloudsource"
     echo "    TESTHEAD: $TESTHEAD"
+    echo "    scenario: $scenario"
```

Comment:
I choose `scenario` as this is PR could also be the very first step to integrate real scenario files to mkcloud. In the corresponding trello cards it says that the crowbar_batch `proposals:` information should be part of a bigger scenario file that ultimately replaces a lot of ENV variables.